<ui version="4.0" >
 <author>Dag Andersen &lt;danders@get2net.dk></author>
 <class>KPlato::TaskProgressPanelBase</class>
 <widget class="QWidget" name="KPlato::TaskProgressPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TaskProgressPanelBase</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="completionTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>275</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Completion</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="started" >
         <property name="text" >
          <string>Started:</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QDateTimeEdit" name="startTime" />
       </item>
       <item row="0" column="2" >
        <widget class="QCheckBox" name="finished" >
         <property name="text" >
          <string>Finished:</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QDateTimeEdit" name="finishTime" />
       </item>
       <item row="0" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="5" >
        <widget class="QGroupBox" name="performedGroup" >
         <property name="title" >
          <string/>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="addEntryBtn" >
              <property name="text" >
               <string>Add Entry</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeEntryBtn" >
              <property name="text" >
               <string>Remove Entry</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="CompletionEntryEditor" name="entryTable" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="usedEffortTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>275</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Per Resource</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="ui_year" >
           <property name="minimum" >
            <number>1900</number>
           </property>
           <property name="maximum" >
            <number>2999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prevWeekBtn" >
           <property name="text" >
            <string>Prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KComboBox" name="weekNumber" />
         </item>
         <item>
          <widget class="QPushButton" name="nextWeekBtn" >
           <property name="text" >
            <string>Next</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>16</width>
             <height>25</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addResource" >
           <property name="text" >
            <string>Add Resource...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="UsedEffortEditor" name="resourceTable" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="optionsTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>275</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Mode</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QRadioButton" name="optionCompleted" >
         <property name="text" >
          <string>Have used- and remaining effort calculated based on percent completed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="optionEffort" >
         <property name="text" >
          <string>Enter used effort per task</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QRadioButton" name="optionResource" >
         <property name="text" >
          <string>Enter used effort per resource</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>UsedEffortEditor</class>
   <extends>QTableView</extends>
   <header>kptusedefforteditor.h</header>
  </customwidget>
  <customwidget>
   <class>CompletionEntryEditor</class>
   <extends>QTableView</extends>
   <header>kptusedefforteditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>prevWeekBtn</tabstop>
  <tabstop>nextWeekBtn</tabstop>
  <tabstop>addResource</tabstop>
  <tabstop>resourceTable</tabstop>
 </tabstops>
 <includes>
  <include location="local" >kdatetimewidget.h</include>
  <include location="local" >kdatewidget.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >kptusedefforteditor.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
